docs/source/release/v6.9.0/diffraction.rst

```diff
@@ -10,59 +10,59 @@ Powder Diffraction
 
 
 Single Crystal Diffraction
 --------------------------
 
 New features
 ############
-- New shoebox integration algorithm for single-crystal Bragg peaks :ref:`IntegratePeaksShoeboxTOF <algm-IntegratePeaksShoeboxTOF>`
-- Add option to clear instrument calibration from peaks workspace in addition to a Workspace2D in ``SXD.undo_calibration``
-- Add documentation for :ref:`ISIS Single Crystal Diffraction Reduction Classes <isis-single-crystal-diffraction-ref>`
-- New algorithm :ref:`SymmetriseMDHisto <algm-SymmetriseMDHisto>` to symmetrise :ref:`MDHistoWorkspaces <MDHistoWorkspace>` by operations in the Laue class of the point group.
+- Added the :ref:`IntegratePeaksShoeboxTOF <algm-IntegratePeaksShoeboxTOF>` algorithm for single-crystal Bragg peaks.
+- Added an option to clear instrument calibration from a peaks workspace in addition to a Workspace2D in ``SXD.undo_calibration``.
+- Added documentation for :ref:`ISIS Single Crystal Diffraction Reduction Classes <isis-single-crystal-diffraction-ref>`.
+- Added the :ref:`SymmetriseMDHisto <algm-SymmetriseMDHisto>` algorithm for symmetrising :ref:`MDHistoWorkspaces <MDHistoWorkspace>` by operations in the Laue class of the point group.
 
 Bugfixes
 ############
-- Fix slow creation of ``CrystalStructure`` object when loading a CIF file with many atoms. A new method to add many scatterers at a time is added from ``CompositeBraggScatterer`` to ``CrystalStructure`` rather than one atom at a time.
-- Fixed typo in argument name in :ref:`WISH reduction class <isis-single-crystal-diffraction-wish-ref>` method ``WishSX.find_sx_peaks`` that was causing method to fail
-- Renamed arguments of ``WishSX.remove_peaks_on_edge`` to be consistent with ``SXD.remove_peaks_on_edge`` (renamed ``nedge_pix`` to and ``nedge_tube`` to ``ntubes``)
-- Fix bug requiring string when workspace object passed to ``BaseSX.convert_ws_to_MD`` (now accepts workspace objects) (``BaseSX`` is the bace class for the :ref:`ISIS Single Crystal Diffraction Reduction Classes <isis-single-crystal-diffraction-ref>`)
-- Set default number density units as formula units per cubic Angstrom in the ISIS single crystal reduction classes (the units can be changed by passing the ``NumberDensityUnit`` argument in the material dictionary in ``set_sample`` - see :ref:`SetSampleMaterial<algm-SetSampleMaterial>` for details).
+- Fixed the slow creation of a ``CrystalStructure`` object when loading a CIF file with many atoms. A new method to add many scatterers at a time is added from ``CompositeBraggScatterer`` to ``CrystalStructure`` rather than one atom at a time.
+- Fixed a typo in the argument name of the :ref:`WISH reduction class <isis-single-crystal-diffraction-wish-ref>` method ``WishSX.find_sx_peaks`` that was causing a method to fail.
+- Renamed arguments of ``WishSX.remove_peaks_on_edge`` to be consistent with ``SXD.remove_peaks_on_edge`` (renamed ``nedge_pix`` to and ``nedge_tube`` to ``ntubes``).
```

Comment:
it looks like the new name for ``nedge_pix`` is missing in this sentence:
`(renamed ``nedge_pix`` to and ``nedge_tube`` to ``ntubes``)`